spec/queries/get_fraud_matches_spec.rb

```diff
@@ -75,5 +75,21 @@
         expect(postcodes).to include('W69BH')
       end
     end
+
+    context 'when duplicated unsubmitted applications' do
+      let(:candidate_ids) { returned_array_of_hashes.map { |element| element['candidate_id'] } }
+
+      before do
+        Timecop.freeze(Time.zone.local(2020, 8, 23, 12, 0o0, 0o0)) do
```

Comment:
`0o0`?

Reply:
(couldn't we just omit these)?

Reply:
`0o0` just zero 😆. Yes agreed! :) Done ♻️